test/fixtures/operator/custom-resource.yaml

```diff
@@ -5,3 +5,10 @@ metadata:
   namespace: snyk-monitor
 spec:
   integrationApi: https://kubernetes-upstream.dev.snyk.io
+  clusterName: My dev clusterName
+  pvc:
+    enabled: false
+    name: snyk-monitor-pvc
+  initContainerImage:
+    repository: busybox
+    tag: latest
```

Comment:
Should we enable this on the OpenShift cluster? If we do, I think we should reduce the disk size to something like 10-15 GB because the cluster was provisioned with a pretty small disk I think 😓 

Also we don't need to list clusterName and initContainerImage, they will use the default values